packages/manager/modules/hub/src/components/products/translations/Messages_de_DE.json

```diff
@@ -3,6 +3,7 @@
   "manager_hub_products_CLOUD_PROJECT": "PCI Projekte",
   "manager_hub_products_CLOUD_DB_ENTERPRISE_CLUSTER": "Enterprise Cloud Database",
   "manager_hub_products_DBAAS_LOGS": "Logs Data Platform",
+  "manager_hub_products_DBAAS_QUEUE": "DbaaS Queue",
```

Comment:
Lot of Updates (https://github.com/ovh/manager/commits/master/packages/manager/modules/hub/src/components/products/translations/Messages_fr_FR.json)